README.md

```diff
@@ -1,37 +1,44 @@
+- Compound
+- CRISPR
+- ORF
 
-## 1.process_data
+Each treatment has it's own platemap and metadata file that can be found in the [reference_plate_data](./reference_plate_data/) folder.
+A barcode platemap is include which associates each plate to the correct platemap file.
 
-In this step, we merged the single cell data using pycytominer, and the reference plate data located in the `reference_plate_data` folder in the root of the git directory.
-The metadata files in this folder can be located by following this aws link:
-`https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0000-jump-pilot/source_4/workspace/metadata/external_metadata/`
+There are a total of **20,959,860 single cells** between all plates.
 
-The platemaps files in this folder can be found using this aws link:
-`https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0000-jump-pilot/source_4/workspace/metadata/platemaps/2020_11_04_CPJUMP1/platemap/`
+To reproduce this project, please ensure adequate storage as the CellProfiler SQLite database files is approximately 1.1 TB large.
 
-Similarly the barcode file `barcode_platemap.csv` came from dropbox.
+## Goal
 
-After merging the single cells, each of the features of the plate data were normalized by pycytominer and converted to parquet files.
+Traditionally, image-based analyses aggregate single-cell features which removes outliers that could be interesting biology.
+By predicting phenotypes for single-cells with the phenotypic profiling model, we hope to uncover patterns of biology that wouldn't be uncovered with the traditional methodology.
 
-## 2.evaluate_data
+## Repository Structure
 
-In this step, we perform inferencing with the pre-trained shuffled and unshuffled logistic regression models from the phenotypic profiling repository `https://github.com/WayScience/phenotypic_profiling_model/tree/main`.
-We generate predicted probabilities for each cell expressing one of the fifteen possible phenotypes.
-This data in addition to the other single-cell metadata is stored as a parquet file.
+| Module | Purpose | Description |
+| :---- | :----- | :---------- |
+| [0.download_data](./0.download_data/) | Download JUMP-Target SQLite files | Download CellProfiler SQLite outputs for 51 plates from AWS |
+| [1.process_data](./1.process_data/) | Process SQLite files | Using pycytominer, the SQLite outputs are converted to CSV features are merged into single-cells and normalized |
+| [2.evaluate_data](./2.evaluate_data/) | Apply phenotypic profiling model | Phenotypic predictions are generated for single-cells using the phenotypic profiling model |
+| [3.analyze_data](./3.analyze_data/) | Analyze phenotypic predictions | Multiple analyses are performed to validate the phenotypic predicted class for each treatment compared to control |
+| [reference_plate_data](./reference_plate_data/) | Platemaps per treatment type | This folder holds the platemap files with metadata based on treatment and the barcode platemap file |
```

Comment:
There is some passive voice in this table, please update to active voice :)

Reply:
I have corrected this! Thanks for pointing it out!